### DISCUSSION TITLE
Copy model config to clipboard

### DISCUSSION THREAD

**Discussion by talmo** [talmolab/sleap#1400] [Ideas]
It's common that we see people take screenshots for sharing model configurations (e.g., #1392, #1397, #1369).

While sharing the JSON of the `TrainingJobConfig` would work, it's not the easiest file to find if you're working from the GUI and not the easiest to quickly upload and share.

It'd be great if we had a button in the training form that copied a simplified set of parameters in plain text or yaml to the clipboard for easy pasting to issues, papers, or your own notes so that you can reuse them.

**Comment by roomrys** [comment]
This was completed in #1433!
